### DISCUSSION TITLE
posts/20240818-cloudflare-with-vercel-setting

### DISCUSSION THREAD

**Discussion by giscus** [abe1272001/abe.dev#22] [Blog Comments]
# posts/20240818-cloudflare-with-vercel-setting

本指南將帶你完成在 Cloudflare 購買域名，並在 Vercel 上使用該域名的整個過程。充分利用 Cloudflare 強大的 DNS 管理和安全功能，以及 Vercel 高效的網站託管服務。

https://abechen.net/posts/20240818-cloudflare-with-vercel-setting

**Comment by chilinglee** [comment]
很棒的分享

**Reply by abe1272001** [reply]
Thanks

**Comment by noah4520** [comment]
推推，Cloudflare 真的很好用

**Reply by abe1272001** [reply]
真的！文件也寫得很好~
